images/tekton/tests/main.tf

```diff
@@ -86,6 +87,7 @@ EOtask
 
 kubectl wait --for=condition=succeeded taskrun test-taskrun --timeout=120s
       EOF
+      retry = { attempts = 5, delay = "10s" }
```

Comment:
```suggestion
```

lets leave it off here, since the kubectl create will conflict with names

Reply:
exactly! I was about to say that I noticed that after I did the change, thanks for noticing this as well.